styles.css

```diff
@@ -28,6 +25,12 @@ body {
   align-items: center;
 }
 
+@media (min-width: 768px) {
```

Comment:
- [ ] The hamburger menu should not be displayed on the desktop view, you can implement the nav links so they can display on the desktop screen size instead of the hamburger menu